lib/assets/index.js

```diff
@@ -1,5 +1,18 @@
 module.exports = {
-  'icon-32': __dirname + '/icon-32.png',
-  'icon-paused-32': __dirname + '/icon-paused-32.png',
-  'icon-playing-32': __dirname + '/icon-playing-32.png'
+	darwin: {
+		icon: __dirname + '/darwin-icon.png',
+		playing: __dirname + '/darwin-icon-playing.png',
+		paused: __dirname + '/darwin-icon-paused.png'
+	},
+	win32: {
+		// Can use darwin icons, because sizes are the same
+		icon: __dirname + '/darwin-icon.png',
+		playing: __dirname + '/darwin-icon-playing.png',
+		paused: __dirname + '/darwin-icon-paused.png'
+	},
+	any: {
+		icon: __dirname + '/icon.png',
+		playing: __dirname + '/icon-playing.png',
+		paused: __dirname + '/icon-paused.png'
+	}
```

Comment:
Sooo, I don't think we need different images for each OS. Instead, it should be different sizes like how we had it before (e.g. `-32`, `-24`). Then, we could have additional properties as you do for each platform which point to those files:

```js
module.exports = {
  'icon-32': __dirname + '/icon-32.png',
  'icon-paused-32': __dirname + '/icon-paused-32.png',
  'icon-playing-32': __dirname + '/icon-playing-32.png'
};

module.exports.darwin = module.exports.win32 = {
  icon: module.exports['icon-32'],
  'icon-playing': module.exports['icon-playing-32'],
  'icon-paused': module.exports['icon-paused-32']
};
```